src/main/java/com/google/sps/utils/UserUtils.java

```diff
@@ -65,4 +65,33 @@ public Entity getEntityFromDatastore(String entityName, String entityPropertyTit
     Entity entity = resultsList.get(0);
     return entity;
   }
+
+   
+   /**
+    * Returns a single Entity object that can then be used in a servlet. 
```

Comment:
u can take out "that can then be used in a servlet". I think that might count as putting stuff about how the servlet should be used as opposed to just what it does in the description. I would also specify that it's a user entity